certdocs/TRAQ-138-SDD.md

```diff
@@ -870,6 +880,14 @@ A sample configuration file is shown below:
             "parent": {
                 "prefix": "TEST",
                 "level": "SWH"
+                "srcAttribute": {
+                    "name": "Item",
+                    "value": "^Item1$"
+                },
+                "dstAttribute": {
+                    "name": "Item Allocation",
+                    "value": "Item1"
+                 }
```

Comment:
I'm not sure I get the idea of what these apply to. Could you give me a practical example of how this could be used?

Reply:
I also find it a bit confusing what src an dst mean in this context (is the src the parent or the child?

Reply:
Well, the links are specified in the context of the child. The child requirements/implementation/test links up to the parent and not the other way round, so if I'm linking **from** something **to** something else then to me the source / destination makes sense to mean that source is the child and destination is the parent.

I suppose I could rename it to 'childAttribute' and 'parentAttribute' but I'm not keen on that, it removes context from the link spec and makes it more confusing (to me). What do you think? Perhaps there's a third option I've not thought of.